cpp/include/cudf/structs/structs_column_view.hpp

```diff
@@ -30,19 +30,40 @@ namespace cudf {
  * @{
  */
 
+/**
+ * @brief Given a column-view of struct type, an instance of this class
```

Comment:
I don't think we ever need to hyphenate "column view"
```suggestion
 * @brief Given a column view of struct type, an instance of this class
```